### PR TITLE
Milestone 1: Setup and mobile first

### PR DESCRIPTION
**Pull request by AHP15**
- Added setup for Webhint, Stylelint, and GitHub actions.
- Added a descriptive README file.
- Added index.html for the toolbar and headline.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Abdessittir Harkati</title>
+</head>
+<body>
+    <main class="main">
+        <header class="header">
+            <h2>Welcome</h2>
+            <button>
+                <img class="menu-icon" src="./media/Union.png" alt="menu" />
+            </button>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] Great job with your project 👍 . Kindly wrap your `hamburger menu` in a `nav` tag 🙏 . The `nav` tag is for semantic reasons.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:wght@300&family=Outfit:wght@300&family=Roboto:ital,wght@0,100;0,300;1,300&family=Water+Brush&display=swap');
+
+@media all and (min-width: 350px) {
+  * {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+    box-sizing: border-box;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly make sure you are using the correct `font-family` as seen in your chosen design 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Abdessittir  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @Abdessittir ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implement✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

*************************************
# Optional Suggestions 

- [ ] N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**